renderer/pages/settings.js

```diff
@@ -78,6 +86,10 @@ const Settings = () => {
                 label={<FormattedMessage id='Settings.Privacy.CollectAnalytics' />}
                 optionKey='advanced.collect_usage_stats'
               />
+              <BooleanOption
+                label={<FormattedMessage id='Settings.Privacy.SendCrashReports' />}
+                optionKey='advanced.send_crash_reports'
+              />
               <BooleanOption
                 label={<FormattedMessage id='Settings.Sharing.UploadResults' />}
                 optionKey='sharing.upload_results'
```

Comment:
While we are at it, we should also remove the option to enable IP address collection.